syntax = "proto2";

message ModifyActionReq {
	required string action_name = 1;
	required int64 execute_order = 2;
	required int64 execute_time = 3;
	required string arguments = 4;
}

message ModifyActionResp {
	required int64 state = 1;
}
